queue_services/entity-emailer/tests/unit/test_worker.py

```diff
@@ -466,3 +469,42 @@ def test_send_email_with_incomplete_payload(app, session, email_msg):
         worker.send_email(email_msg, None)
 
     assert 'Unsuccessful sending email' in str(excinfo)
+
+
+def test_ar_overdue_stage_1_notification(app, session, mocker):
+    """Assert that the stage 1 overdue ARs notification can be processed."""
+    token = 'token'
+    business_identifier = 'BC1234567'
+    business = create_business(business_identifier, 'BC', 'Test Business')
+    furnishing = create_furnishing(business)
+    
+    mocker.patch(
+        'entity_emailer.email_processors.ar_overdue_stage_1_notification.get_jurisdictions',
+        return_value=[])
+
+    # run worker
+    with patch.object(AccountService, 'get_bearer_token', return_value=token):
+        with patch.object(worker, 'send_email', return_value='success') as mock_send_email:
+            with patch.object(ar_overdue_stage_1_notification, 'get_recipient_from_auth', return_value='test@test.com'):
```

Comment:
don't think this is required anymore if we get the email recipient from the furnishings record

Reply:
updated